README.md

```diff
@@ -454,12 +454,11 @@ Future<int> deletePersons(List<Person> person);
 ```
 
 ### Streams
-As already mentioned, queries can not only return a value once when called but also a continuous stream of query results.
-The returned stream keeps you in sync with the changes happening to the database table.
-This feature plays really well with the `StreamBuilder` widget.
+As already mentioned, queries cannot only return values once when called but also continuous streams of query results.
```

Comment:
This "cannot" seems weird here... I think it was better before.
But I'm not a native, so I don't really know.

Reply:
My IDE suggested that change. Apparently "cannot" is used more frequently compared to "can not". https://www.grammarly.com/blog/cannot-or-can-not/

Reply:
I would usually agree, but the combination with "only" is what triggered me... 
well, it's not that important :smile:

Reply:
*evil laugh* found something confirming my position: https://english.stackexchange.com/questions/421944/can-we-use-the-wording-we-can-not-only-but-also

Reply:
Haha I see 😄